content/docs/command-reference/params/index.md

```diff
@@ -143,6 +143,61 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Python parameters file
+
+Consider this parameters file in Python format, named `params.py`:
+
+```python
+IS_BOOL: bool = True
+CONST = 5
+
+
+class TrainConfig:
+    EPOCHS = 70
+
+    def __init__(self):
+        self.layers = 9
+
+
+class TestConfig:
+    TEST_DIR = "path"
+    METRICS = ["metric"]
+```
+
+The following [stage](/doc/command-reference/run) depends on params
+`IS_BOOL`, `CONST`, as well as `TrainConfig`'s `EPOCHS` and `layers`:
+
+```dvc
+$ dvc run -n train -d users.csv -o model.pkl \
+          -p params.py:IS_BOOL,CONST,TrainConfig.EPOCHS,TrainConfig.layers \
+          python train.py
+```
+
+Resulting `dvc.yaml` file (notice the `params` list):
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - users.csv
+    params:
+      - IS_BOOL
+      - CONST
+      - TrainConfig.EPOCHS
+      - TrainConfig.layers
+    outs:
+      - model.pkl
+```
+
+Alternatively, the entire group of parameters `TestConfig` can be referenced:
+
+```dvc
+$ dvc run -n train -d users.csv -o model.pkl \
+          -p params.py:TestConfig \
+          python train.py
+```
```

Comment:
```suggestion
Alternatively, the entire `TestConfig` group can be referenced, instead of the
parameters in it:

```dvc
$ dvc run -n train -d users.csv -o model.pkl \
          -p params.py:IS_BOOL,CONST,TestConfig \
          python train.py
```
```